swagger/EndpointDocumentation/Projects/project_w_id.yml

```diff
@@ -0,0 +1,76 @@
+put:
```

Comment:
Separate these per route. So there's one file for each route. So like `create.yml`, `update.yml`, etc. to keep it consistent with the users thing

Reply:
I'm realizing this may not be possible with the way openAPI is setup. If it is possible, that's what I would recommend. Otherwise, not a big deal